### ISSUE TITLE
[2.7.0-RC8] "The method render() is ambiguous" error in Eclipse

### ISSUE THREAD

**Issue by slisaasquatch** [playframework/playframework#8910]
### Play Version (2.5.x / etc)
2.7.0-RC8 (This time I can confirm it's actually RC8)

### API (Scala / Java / Neither / Both)
Java

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)
Windows 10

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)
Oracle 1.8.0_181

### Library Dependencies
None

### Actual Behavior
I have `index.scala.html` with:
```html
@()
<!doctype html>

<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Play REST API</title>
  </head>

  <body>
    <h1>Play REST API</h1>
  </body>
</html>
```
I'm calling it like this in my controller:
```java
	public Result index() {
		return ok(com.example.views.html.index.render());
	}
```
`sbt compile` and `sbt eclipse` complete with no errors. Also, my app runs completely fine.
But after I refresh my project in Eclipse, I'm getting an error in my controller saying "The method render() is ambiguous for type index". If I add any parameter to the template, the error goes away.

### Reproducible Test Case
None

**Comment by mkurz** [comment]
So that is "just" a problem within Eclipse? The app however works correctly?

**Comment by slisaasquatch** [comment]
> So that is "just" a problem within Eclipse? The app however works correctly?

That is correct.

**Comment by TimMoore** [comment]
This sounds like this Scala compiler bug: https://github.com/scala/bug/issues/11271

It's fixed in Scala 2.12.8. I _think_ that updating `scalaVersion` in your project should be enough to fix it for Twirl templates (because the `render` method is generated).

**Comment by slisaasquatch** [comment]
> This sounds like this Scala compiler bug: [scala/bug#11271](https://github.com/scala/bug/issues/11271)
> 
> It's fixed in Scala 2.12.8. I _think_ that updating `scalaVersion` in your project should be enough to fix it for Twirl templates (because the `render` method is generated).

You are right. I changed my `build.sbt` to have this `scalaVersion := "2.12.8"` and that fixed it. I think Play 2.7 sample projects should be updated with this as well, because that's the only reason I was using 2.12.7. I'll close this ticket.
